org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/ContentModelCodeLensParticipant.java

```diff
@@ -105,6 +105,10 @@ private static boolean canSupportOpenUri(ICodeLensRequest request) {
 
 	private CodeLens createReferencedGrammarLens(ReferencedGrammarInfo info, Range range, boolean canSupportOpenUri) {
 		String uri = info.getIdentifierURI();
+		if (uri == null) {
+			// when using file associations to bind, identifier URI is not set
+			uri = info.getResolvedURIInfo().getResolvedURI();

```

Comment:
I would prefer to set empty value when uri is null, because resolved URI can be long (cache file path).

In the case of file association we should display the pattern of file associaton but it's not possible today, so please create an issue for that.

Reply:
https://github.com/eclipse/lemminx/issues/1101